### DISCUSSION TITLE
📲

### DISCUSSION THREAD

**Discussion by CM131T** [community/community#38907] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.88.0 (82868297)
Device: iPhone 12 Pro Max (iOS 16.1.1)
</details>

**Comment by jfullstackdev** [comment]
you did not mention about the bug, what is the bug ?

**Comment by ghostinhershell** [comment]
Hi @CM131T, welcome, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails and what questions you have. Giving more details might help someone give you a nudge in the right direction.
